src/OnboardingManager.tsx

```diff
@@ -34,6 +35,8 @@ const useInitializeAfterOnboarding = (onboardingComplete: boolean) => {
       if (pushNotificationsActive) {
         Initializers[Initializer.PushNotifications]();
       }
+
+      ScreenOrientation.lockAsync(ScreenOrientation.OrientationLock.DEFAULT);
```

Comment:
you removed the comment here, which could somewhen be helpful, as the method is not very self explanatory.
otherwise one need to know what the lock does and what DEFAULT stands for.

> // set orientation to "default", to allow both portrait and landscape

Reply:
added it again